### DISCUSSION TITLE
Seeing error ` when using contrib: exporters/metric/dogstatsd e

### DISCUSSION THREAD

**Discussion by luck02** [open-telemetry/opentelemetry-go#1546] [Q&A] [answered]
error: 

```
../../go/pkg/mod/go.opentelemetry.io/contrib/exporters/metric/dogstatsd@v0.16.0/dogstatsd.go:78:2: undefined: otel.SetMeterProvider
```

The code is mostly based on:
https://github.com/open-telemetry/opentelemetry-go-contrib/blob/main/exporters/metric/dogstatsd/example_test.go 

Which i see references:
```
require (
	github.com/stretchr/testify v1.7.0
	go.opentelemetry.io/otel v0.16.0
	go.opentelemetry.io/otel/sdk v0.16.0
)
```

So presumably this is because I started with otel v0.17.0, if that's correct then:  

* Is best course to revert to otel 0.16.0
* Any idea when this'll get upgraded to 0.17.0?  

Thanks!

**Comment by MrAlias** [comment, marked as answer]
The [v0.17.0 Release](https://github.com/open-telemetry/opentelemetry-go-contrib/releases/tag/v0.17.0) of the opentelemetry-go-contrib project was made a few hours ago. Please give that a try.
